src/digarch_scripts/transfer/transfer_rsync.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import logging
+import re
+import subprocess
+from pathlib import Path
+
+import digarch_scripts.package.package_base as pb
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+
+
+def parse_args() -> argparse.Namespace:
+    parser = pb.TransferParser(
+        description="Create packages for all file transfer files for a single acquisition."
+    )
+    parser.add_carrierid()
+    parser.add_source()
+    parser.add_dest()
+    parser.add_quiet(help="Suppresses progress bar from rsync")
+
+    return parser.parse_args()
+
+
+def run_rsync(source: Path, dest: Path, quiet: bool = None) -> None:
```

Comment:
currently rsync seems to be unable to transfer some files. This will throw an error but won't indicate what files aren't transferred or why. There may be additional rsync flags to add to this function.

Reply:
add a logger to check the log for what didn't transfer